mlrun/db/httpdb.py

```diff
@@ -2623,8 +2634,8 @@ def list_model_endpoints(
         :param project: The name of the project
         :param model: The name of the model to filter by
         :param function: The name of the function to filter by
-        :param labels: A list of labels to filter by. Label filters work by either filtering a specific value of a label
-            (i.e. list("key==value")) or by looking for the existence of a given key (i.e. "key")
+        :param labels: A list of labels to filter by. Label filters work by either filtering a specific value of a
+         label (i.e. list("key==value")) or by looking for the existence of a given key (i.e. "key")
```

Comment:
```suggestion
         label (i.e. list("key=value")) or by looking for the existence of a given key (i.e. "key")
```
In the backend you split by a single `=`.